installers/lattepanda-alpha/preseed.cfg

```diff
@@ -59,14 +66,15 @@ d-i partman-auto/expert_recipe string        \
                       format{ }              \
                       use_filesystem{ }      \
                       filesystem{ ext4 }     \
+                      device{ /dev/mmcblk0 } \
                       mountpoint{ /boot }    \
               .                              \
               40960 40960 -1 ext4            \
                       $defaultignore         \
                       $primary{ }            \
                       label{ avena-lvm }     \
                       method{ lvm }          \
-                      device{ /dev/sda }     \
+                      device{ /dev/sda } \
```

Comment:
Does this whitespace cause any issues? I know partman can be a butt head.

Reply:
Better safe than sorry